driver-core/src/test/java/com/datastax/driver/core/DataTypeCqlNameParserTest.java

```diff
@@ -88,8 +89,8 @@ public void should_parse_nested_collection_types() {
         assertThat(parse("list<list<int>>", cluster, null, null, null, false, false)).isEqualTo(list(list(cint())));
         assertThat(parse("set<list<frozen<map<bigint,varchar>>>>", cluster, null, null, null, false, false)).isEqualTo(set(list(map(bigint(), varchar(), true))));
 
-        UserType keyType = keyspaceMetadata.getUserType("Incr,edibly\" EvilTy<>><<><p\"e");
-        UserType valueType = keyspaceMetadata.getUserType("\"A\"");
+        UserType keyType = keyspaceMetadata.getUserType(quote("Incr,edibly\" EvilTy<>><<><p\"e"));
```

Comment:
This shows that `quote` method is now behaving well. It's odd that it also works without calling `quote` and simply passing the identifier, but there is a comment about it in `handleId` so I guess it's a known corner case.